src/Eloquent/Builder.php

```diff
@@ -30,24 +30,33 @@ class Builder extends EloquentBuilder
         'avg',
         'count',
         'dd',
-        'doesntExist',
+        'doesntexist',
         'dump',
         'exists',
-        'getBindings',
-        'getConnection',
-        'getGrammar',
+        'getbindings',
+        'getconnection',
+        'getgrammar',
         'insert',
-        'insertGetId',
-        'insertOrIgnore',
-        'insertUsing',
+        'insertgetid',
+        'insertorignore',
+        'insertusing',
         'max',
         'min',
         'pluck',
         'pull',
         'push',
         'raw',
         'sum',
-        'toSql',
+        'tomql',
+        // Kept for compatibility with Laravel < 10.3
```

Comment:
And here I was trying to conditionally set the method names casing 😆

Reply:
Simple and verbose solutions are often good enough.